src/Symfony/Component/Serializer/Tests/Encoder/ChainEncoderTest.php

```diff
@@ -84,42 +84,13 @@ public function testNeedsNormalizationBasic()
         $this->assertTrue($this->chainEncoder->needsNormalization(self::FORMAT_2));
     }
 
-    /**
-     * @dataProvider booleanProvider
-     */
-    public function testNeedsNormalizationChainNormalizationAware($bool)
```

Comment:
This case relies on `ChainNormalizationAwareEncoder`, which extends `ChainEncoder`, but the class is `@final` so cannot be extended. A fix on phpunit-bridge is coming on branch 4.4 to trigger the missing deprecation.